decidim/decidim
# Implement breadcrumb menu in all admin pages

#### :tophat: What? Why?

There were some pages without breadcrumb in the admin. This PR fixes it.

Apart from that, I found a couple mini bugs:

- We were using the `components-counter--off` in the HTML but it didn't have any CSS 
- The components counter in the Global Moderations is broken, as it didn't correspond to the ones that we have in components. (Just thinking out loud that we should rename it to `resources-counter` or something more generic?)

#### :pushpin: Related Issues

- Fixes #11387

#### Testing

1. Visit different pages in the admin panel
2. See that the breadcrumb corresponds to the page that you're at 

:hearts: Thank you!
